flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/utils/TypeUtils.java

```diff
@@ -126,13 +126,19 @@ private static void addTypeRefIds(TypeSymbol ts, ModuleInfo moduleInfo, List<Str
         }
     }
 
+    /**
+     * Check whether the given symbol is a http module.
+     *
+     * @param symbol symbol to check
+     * @return true if the symbol is a http module, false otherwise
+     */
     public static boolean isHttpModule(Symbol symbol) {
         Optional<ModuleSymbol> module = symbol.getModule();
         if (module.isEmpty()) {
             return false;
         }
 
         ModuleID moduleId = module.get().id();
-        return moduleId.orgName().equals("ballerina") && moduleId.packageName().equals("http");
+        return moduleId.orgName().equals(CommonUtils.BALLERINA_ORG_NAME) && moduleId.packageName().equals("http");
     }
```

Comment:
Do we need this function? Can't we use `isWithinPackage(Symbol symbol, ModuleInfo moduleInfo)` for the same usecases?

Reply:
Here we need specifically check wether this symbol belongs to ballerina/http module.